Dockerfile

```diff
@@ -10,7 +10,6 @@ RUN apt-get update && \
                     curl \
                     bzip2 \
                     ca-certificates \
-                    xvfb  \
```

Comment:
I just spotted this, sorry for being late. Why do we keep the python `xvfbwrapper` but stop installing xvfb?

Reply:
We do install it. I moved it to the afni block below.